infra/core/host/enrichmentapp/enrichmentapp.tf

```diff
@@ -103,6 +103,7 @@ resource "azurerm_linux_web_app" "app_service" {
       "AZURE_SEARCH_SERVICE_KEY"                  = "@Microsoft.KeyVault(SecretUri=${var.keyVaultUri}secrets/AZURE-SEARCH-SERVICE-KEY)"
       "COSMOSDB_KEY"                              = "@Microsoft.KeyVault(SecretUri=${var.keyVaultUri}secrets/COSMOSDB-KEY)"
       "ENRICHMENT_KEY"                            = "@Microsoft.KeyVault(SecretUri=${var.keyVaultUri}secrets/ENRICHMENT-KEY)"
+      "BING_SEARCH_KEY"                            = "@Microsoft.KeyVault(SecretUri=${var.keyVaultUri}secrets/BINGSEARCH-KEY)"
```

Comment:
The enrichment app does not need this key.

Reply:
Removed. Incoming Changes